plugins/rtmp-services/data/package.json

```diff
@@ -1,10 +1,5 @@
 {
-	"url": "https://obsproject.com/obs2_update/rtmp-services",
-	"version": 132,
-	"files": [
-		{
-			"name": "services.json",
-			"version": 132
-		}
-	]
+	"url" : "https://obsproject.com/obs2_update/rtmp-services",
```

Comment:
Is the formatting change produced by clang-format?

Reply:
I ran clang-format -i package.json on the file, so probably? but I can check...also when I run this I see
![image](https://user-images.githubusercontent.com/60421398/83559318-63136100-a4e2-11ea-8e6c-f1d4d9c4f82e.png)